src/components/recs/team-tile.tsx

```diff
@@ -1,12 +1,13 @@
 import { teamIndexToPlayerData } from "@/server/teams";
-import { majorGodIndexToData } from "@/types/MajorGods";
+import { majorGodIndexToData, majorGodNameToData } from "@/types/MajorGods";
```

Comment:
Is majorGodNameToData being used?

Reply:
I also forgot to undo this when undoing the frontend display change - will sort this as well